death_note.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "sync"
+
+type DeathNote struct {
+	paramSet map[string]bool
+	rwMutex sync.RWMutex
```

Comment:
can we call this a `lock`?

```suggestion
	lock sync.RWMutex
```

I think that's a common thing to name it like that